src/platform/PICO/dshot.pio

```diff
@@ -19,13 +19,15 @@
 ;
 ; All programs, bit width = 40 cycles, clock set accordingly
 ; Min delay between frames of 2us, number of cycles varies according to program
-;
+
 ; DSHOT 600
 ; 1 bit ~ 1.667us, 2us ~ 48 cycles. Allow say 56 cycles between frames
 ;
-
 ; * could get rid of nop by adding delays to 2x jmp !osre, bitloop
 ; * maybe get rid of discard by writing 16-bit word (dupllicated to top 16 bits), and osre empty after 16 bits
+;
+; could be simpler to let the PIO "bitbang" bytes of 11111100 or 11100000 (which have the correct ratios
+; of 6:2 and 3:5 to represent 1 and 0). The 4-word TX fifo has enough space (4x32 bits = 16 bytes) for the 16 dshot bits required.
 
 .program dshot_600
```

Comment:
DShot isn't that different from ledstrip protocol. Maybe some ledstrip tricks can be applied here?